projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -3,28 +3,32 @@
 ### This is a list detailing changes for all Jetpack releases.
 
 ## 10.8-a.9 - 2022-03-23
+### Major Enhancements
+- QR Post: add new feature which automatically generates QR codes for published posts. When scanned, the QR code will link visitors to the post. If a site has a custom logo set, it will be shown in the generated QR code image.
+
 ### Enhancements
 - External Media: add Openverse as an external Media provider.
-- Jetpack: open the QR Post feature to public, and improve its logo sizing.
 - Payment Block: clicking on payment links from email or WordPress.com Notifications/Reader will correctly open the corresponding payment form.
-- Search: show search benefits on the Jetpack disconnection screen.
+- Dashboard: show Search benefits on the Jetpack disconnection screen.
 
 ### Improved compatibility
-- Connection: Jetpack now relies on the Connection package webhooks for 'authorize' and 'authorize_redirect' actions.
 - Sharing: avoid warnings when the feature is not active on a site that uses the AMP plugin.
 
 ### Bug fixes
 - Calendly Block: ensure it can be displayed inline when using a block enabled theme.
-- VideoPress: fixed broken styles on resumable uploader component.
+- VideoPress: fix broken styles on resumable uploader component.
 - Pay with Paypal Block: fix inconsistent currency formatting.
 - WordAds: fix defaulting to "house ad" mode for new activations.
 - Publicize: prevent newlines from being stripped from a custom Publicize message in the classic editor.
+- QR Post: improve the inner logo sizing.
 - Shortlinks: display the shortlinks interface in the block editor for all post types that support shortlinks.
 - Top Posts Widget: display a fallback list of posts to admins when there are no popular posts to display.
+- Premium Content Block: fix a Redux store middleware regression on older Gutenberg versions.
```

Comment:
Since that block is only available on WoA and wpcom simple sites, I would move this to "Other".